src/components/dropdown-tag-menu.js

```diff
@@ -21,25 +20,31 @@ const DropdownTagsMenu = (props) => {
     return (
       <Dropdown.Item
         text={object.value}
+        className='ui dropdown menu> item'
         onClick={toggleSelectionDrop}
       >
-        <Checkbox
-          label={object.value}
-          checked={activeTags.includes(object.value)}
-        />
+        <div className="ui attached icon form" id="info_input_form">
```

Comment:
same here you should be using `Form.Input` component, not making a div and styling it with semantic ui